### ISSUE TITLE
Monix regression in 2.12.x build

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#507]
https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1375/consoleFull shows:

```
[monix] Running "test" in: evalJVM
[monix] [info] Compiling 48 Scala sources to /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.7-RC1/project-builds/monix-fde0fbe7c233eef5775a7278f8d18a2c5297415f/monix-eval/jvm/target/scala-2.12/test-classes...
[monix] [error] missing or invalid dependency detected while loading class file 'Asserts.class'.
[monix] [error] Could not access term universe in class scala.reflect.macros.blackbox.Context,
[monix] [error] because it (or its dependencies) are missing. Check your build definition for
[monix] [error] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[monix] [error] A full rebuild may help if 'Asserts.class' was compiled against an incompatible version of scala.reflect.macros.blackbox.Context.
[monix] [error] one error found
[monix] [error] (evalJVM/test:compileIncremental) Compilation failed
```

@alexandru (nice to meet you the other day!) any idea what's up here? the failure was with monix/monix@56240d928737f0a119f8773d61a119ea953a7061, last green build was run 1372 with monix/monix@bfb9274fcf29ddb93bcd0ca8684570a3d8a11ea3 (Scala version in both is 2.12.2-bin-4dc194d)

**Comment by alexandru** [comment]
@SethTisue sorry, for some reason my [Minitest](https://github.com/monix/minitest) needs to depend on `scala-reflect` (compile, not provided) and when I upgraded some dependencies `scala-reflect` was dropped.

I have released Minitest 1.0.1 on Maven Central and the problem should be solved soon.

**Comment by SethTisue** [comment]
fixed since run 1376, thanks!
